astroplan/core.py

```diff
@@ -1512,12 +1788,32 @@ def __repr__(self):
         fmt_coord = repr(self.coord).replace('\n   ', '')[1:-1]
         return '<{} "{}" at {}>'.format(class_name, self.name, fmt_coord)
 
+    @classmethod
+    def _fixed_target_from_name_mock(cls, name):
+        """
+        Mock method to replace `FixedTarget.from_name` in tests.
+        """
+        if name.lower() == "rigel":
```

Comment:
Maybe put the "data" in a list of tuples or some other Python data structure to reduce the boilerplate code a little bit?
```python
targets = [
    ("rigel", 78.63446707, -8.20163837),
    ...
]
```